app/experimenter/nimbus-ui/src/components/ContainerEditPage/index.test.tsx

```diff
@@ -4,11 +4,16 @@
 
 import React from "react";
 import { render, screen, waitFor } from "@testing-library/react";
-import ContainerEditPage from ".";
-import { RouterSlugProvider } from "../../lib/test-utils";
+import ContainerEditPage, { POLL_INTERVAL } from ".";
+import { renderWithRouter, RouterSlugProvider } from "../../lib/test-utils";
 import { mockExperimentQuery } from "../../lib/mocks";
+import { NimbusExperimentStatus } from "../../types/globalTypes";
+import { BASE_PATH } from "../../lib/constants";
+import { act } from "react-dom/test-utils";
```

Comment:
Did VS Code auto import this? I think this `act` should come from `@testing-library/react`

Reply:
Ah yes, the woes of auto-import, nice catch.